extensions/wikia/RTE/RTE.class.php

```diff
@@ -219,7 +223,7 @@ public static function makeGlobalVariablesScript(&$vars) {
 		}
 
 		// initial CK mode (wysiwyg / source)
-		$vars['RTEInitMode'] = self::$initMode;
+		$vars['RTEInitMode'] =  (self::$initMode == self::INIT_MODE_WYSIWYG) ? 'wysiwyg' : 'source';
```

Comment:
I see an opportunity for further improvements here: let's consider using `wysiwyg` and `source` as values of the named constants and get rid of numeric values completely.